financial-connections/src/main/java/com/stripe/android/financialconnections/domain/ErrorHandler.kt

```diff
@@ -0,0 +1,51 @@
+package com.stripe.android.financialconnections.domain
+
+import com.stripe.android.core.Logger
+import com.stripe.android.financialconnections.analytics.FinancialConnectionsAnalyticsTracker
+import com.stripe.android.financialconnections.analytics.logError
+import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest
+import com.stripe.android.financialconnections.navigation.Destination
+import com.stripe.android.financialconnections.navigation.NavigationManager
+import com.stripe.android.financialconnections.repository.FinancialConnectionsErrorRepository
+import javax.inject.Inject
+
+internal class ErrorHandler @Inject constructor(
```

Comment:
Should we create this as an interface and a `RealErrorHandler` to make testing easier?

Reply:
https://github.com/stripe/stripe-android/pull/7892/commits/77c161f866957efbed7c8cc102fe7b55de8649b3